### PR TITLE
Add reference to the blog that shows how to install control plane on ECS

### PR DESCRIPTION
**Pull request by ffjlabo**
**What this PR does / why we need it**:

Added the link for [PipeCD best practice 02 - control plane on ECS](https://pipecd.dev/blog/2023/02/07/pipecd-best-practice-02-control-plane-on-ecs/)

Cursor_と_Install_Control_Plane___PipeCD

Installing_Control_Plane_on_ECS___PipeCD

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**: no
- **Is this breaking change**: no
- **How to migrate (if breaking change)**: no

### CODE REVIEW

**Comment by ffjlabo** [comment]
Added for `dev` and `v0.45.x`

File: docs/content/en/docs-dev/installation/install-control-plane/installing-controlplane-on-ECS.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+linkTitle: "Installing Control Plane on ECS"
+weight: 2
+description: >
+  This page describes how to install control plane on ECS.
+---
+
+Currently, we provide the example of deploying Control Plane to ECS using terraform. 
+
+Please refer to the blog post :)
+[PipeCD best practice 02 - control plane on ECS]({{< ref "/blog/control-plane-on-ecs.md" >}} "PipeCD best practice 02 - control plane on ECS").
```
Comment (ffjlabo) [inline docs/content/en/docs-dev/installation/install-control-plane/installing-controlplane-on-ECS.md:12]: Checked whether to work the link and it is OK ✅ 
ref: https://gohugo.io/content-management/cross-references/#use-of-ref-and-relref

**Review by t-kikuc — APPROVED** [review]
LGTM 🚀

**Review by khanhtc1202 — APPROVED** [review]
Nice

**Pull request merged**